### PR TITLE
Re-voiced and added missing descriptions for some resources, and re-formatted table.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

File: api-reference/beta/resources/webhooks.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ Using the Microsoft Graph API, an app can subscribe to changes on the following
...
-| Teams [chatmessage](/graph/api/resources/subscription?view=graph-rest-beta) | `/teams/allMessages` (messages in all channels in all teams)<br>`/teams/{id}/channels/{id}/messages` (messages in a specific channel)<br>`/chats/allMessages` (messages in all chats)<br>`/chats/{id}/messages` (messages in a specific chat) | Yes |
+| Outlook [message][] | Changes to all messages in a user's mailbox: <br>`/users/{id}/messages`<br>Changes to messages in a user's Inbox:<br>`/users/{id}/mailFolders('inbox')/messages` | No |
+| Outlook [event][] | Changes to all events in a user's mailbox:<br>`/users/{id}/events` | No |
+| Outlook personal [contact][] | Changes to all personal contacts in a user's mailbox:<br>`/users/{id}/contacts` | No |
+| [user][] | Changes to all users:<br>`/users` <br>Changes to a specific user:<br>`/users/{id}`| No |
+| [group][] | Changes to all groups:<br>`/groups` <br>Changes to a specific group:<br>`/groups/{id}` | No |
+| Office 365 group [conversation][] | Changes to a group's conversations:<br>`groups/{id}/conversations` | No |
+| [driveItem][] on OneDrive (personal) | Changes to content within the hierarchy of _any folder_:<br>`/users/{id}/drive/root` | No |
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/webhooks.md]: @baywet , in line 26 and 27, I changed references to /me/drive/root to the more general /users/{id}/root. Would you like to confirm if that's correct?

File: api-reference/v1.0/resources/subscription.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ doc_type: resourcePageType
 
 A subscription allows a client app to receive notifications about changes to data in Microsoft Graph. Currently, subscriptions are enabled for the following resources:
 
-- A [message][], [event][], or [contact][] in Outlook
-- A [conversation][] of an Office 365 group
+- An [alert][] from the Microsoft Graph Security API
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/resources/subscription.md]: Reordering this growing bullet list in alphabetical order, by the resource name. Similar to what's been done in the beta version.

File: api-reference/v1.0/resources/subscription.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ The resource path expressions supported for each resource - that can be used in
 |:---------|:-----|:------------|
 | changeType | string | Required. Indicates the type of change in the subscribed resource that will raise a notification. The supported values are: `created`, `updated`, `deleted`. Multiple values can be combined using a comma-separated list.<br><br>Note: Drive root item notifications support only the `updated` changeType. User and group notifications support `updated` and `deleted` changeType. |
 | notificationUrl | string | Required. The URL of the endpoint that will receive the notifications. This URL must make use of the HTTPS protocol. |
-| resource | string | Required. Specifies the resource that will be monitored for changes. Do not include the base URL (`https://graph.microsoft.com/v1.0/`). |
+| resource | string | Required. Specifies the resource that will be monitored for changes. Do not include the base URL (`https://graph.microsoft.com/v1.0/`). See the possible resource path [values](webhooks.md) to specify for each supported resource.|
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/resources/subscription.md]: Complementing what's added in line 20.

File: api-reference/beta/resources/webhooks.md [hunk 4]
```diff
@@ -17,16 +17,16 @@ Using the Microsoft Graph API, an app can subscribe to changes on the following
...
-| [group][] | `/groups` (changes to all groups)<br>`/groups/{id}` (changes to a specific group) | No |
-| Office 365 group [conversation][] | `groups/{id}/conversations` | No |
-| Content within the hierarchy of _any folder_ [driveItem][] on a user's personal OneDrive | `/me/drive/root` | No |
-| Content within the hierarchy of the _root folder_ [driveItem][] on OneDrive for Business | `/drives/{id}/root`<br> `/me/drive/root` | No |
-| Security [alert][] | `/security/alerts/{id}` (changes to a specific alert) <br> `/security/alerts/?$filter` (changes to filtered alerts)| No |
-| Teams [chatmessage](/graph/api/resources/subscription?view=graph-rest-beta) | `/teams/allMessages` (messages in all channels in all teams)<br>`/teams/{id}/channels/{id}/messages` (messages in a specific channel)<br>`/chats/allMessages` (messages in all chats)<br>`/chats/{id}/messages` (messages in a specific chat) | Yes |
+| Outlook [message][] | Changes to all messages in a user's mailbox: <br>`/users/{id}/messages`<br>Changes to messages in a user's Inbox:<br>`/users/{id}/mailFolders('inbox')/messages` | No |
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/webhooks.md]: Lines 20-26: Making the middle column more scannable.

File: api-reference/beta/resources/subscription.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ The resource path expressions supported for each resource - that can be used in
...
 | notificationUrl | string | The URL of the endpoint that receives the notifications. This URL must make use of the HTTPS protocol. Required. |
 | lifecycleNotificationUrl | string | The URL of the endpoint that receives lifecycle notifications, including `subscriptionRemoved` and `missed` notifications. If not provided, those notifications will be delivered to **notificationUrl**. This URL must make use of the HTTPS protocol. Optional. <br><br>[Read more](/graph/webhooks-outlook-authz) about how Outlook resources use lifecycle notifications. |
-| resource | string | Specifies the resource that will be monitored for changes. Do not include the base URL (`https://graph.microsoft.com/beta/`). Required. |
+| resource | string | Specifies the resource that will be monitored for changes. Do not include the base URL (`https://graph.microsoft.com/beta/`). See the possible resource path [values](webhooks.md) to specify for each supported resource. Required. |
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/subscription.md]: Line 42: Complementing what's added in line 23.

File: api-reference/beta/resources/webhooks.md (hunk 1 above)
Comment (daspek) [reply api-reference/beta/resources/webhooks.md]: It's correct. `/me/drive/root` also works, but up to you on whether it needs to be included.

Comment (baywet) [reply api-reference/beta/resources/webhooks.md]: this aligns with what we have for messages and calendar. Thank you for making the changes! And we also have a line below stating that all paths starting with /users/{id} can start with /me instead.

I thought /drives/{id} should also be listed as we can have items in a SharePoint site collection, maybe /groups/{id}/drive as well? do we want to be exhaustive here or simply list out the trivial cases?

Comment (angelgolfer-ms) [reply api-reference/beta/resources/webhooks.md]: @daspek , thanks for the confirmation. We'll let the note at the end of the table capture all the /me cases and not list them explicitly.
@baywet , I'll let @daspek and you decide.

**Pull request merged**